core/config/project_settings.cpp

```diff
@@ -1337,7 +1337,7 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF(PropertyInfo(Variant::INT, "display/window/size/window_height_override", PROPERTY_HINT_RANGE, "0,4320,1,or_greater"), 0); // 8K resolution
 
 	GLOBAL_DEF("display/window/energy_saving/keep_screen_on", true);
-	GLOBAL_DEF("display/window/energy_saving/keep_screen_on.editor", false);
+	GLOBAL_DEF("display/window/energy_saving/keep_screen_on.editor_main", false);
```

Comment:
This changes behavior a little since the project running from the editor will now keep the screen on (this may or may not be desired, especially on a laptop on battery). Just noting this for the future.

Reply:
I think the previous behavior was not really intended; it came from a limitation.